manifests/params.pp

```diff
@@ -17,21 +17,29 @@
   $version = 'latest'
   $service = 'jenkins'
   $install_from_git = false
+  $install_from_pip = true
+  $install_from_pkg = false
   $git_revision     = 'master'
   $git_url          = 'https://git.openstack.org/openstack-infra/jenkins-job-builder'
 
   case $::osfamily {
     'RedHat', 'Amazon': {
-      $python_packages = [ 'python', 'python-devel', 'python-pip', 'python-argparse']
+      if $::operatingsystemrelease =~ /^6/ {
+        $python_packages = [ 'python', 'python-devel', 'python-pip', 'python-argparse', 'PyYAML']
+        # This requires the dcaro/jenkins-job-builder repository
+        $jjb_packages    = [ 'jenkins-job-builder']
+      } else {
+        $python_packages = [ 'python', 'python-devel', 'python-pip', 'PyYAML']
```

Comment:
Hmm.. I'm also not sure it's worth it - it's unlikely that there are many affected by the issue I saw - I only experienced it because I had set a dependency relationship in our profile module - any of those sorts of issues should be fairly easy to spot. @daenney / @gregswift - all my tests look to be passing with these changes - good to merge from my perspective